src/app/db/db.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore, AngularFirestoreCollection, AngularFirestoreDocument } from 'angularfire2/firestore';
+import { Observable } from 'rxjs/Observable';
+
+import { User } from './db-data';
+
+@Injectable()
+export class DbService {
+
+  users: Observable<User[]>;
+
+  constructor(private angularFirestore: AngularFirestore) {
+    this.users = this.angularFirestore.collection('users').valueChanges();
+  }
+
```

Comment:
It would be cool to have method stubs, returning some mock data: 
 - getCurrentUserName():String;
- isUserAuthorized(): boolean;

These methods're gonna be used in #7 _Add pages header_ issue.